client/app/profile/profile.js

```diff
@@ -1,132 +1,155 @@
+            })
+    }
+
+    $scope.loadCollections = function() {
+        Collections.getAll().then(function(collections){
+          $scope.collections= collections;
+        })
     }
-  });
-};
-
-$scope.photoOpen = function() {
-  var modalInstance = $uibModal.open({
-    templateUrl: "app/profile/photo.html",
-    controller: PhotoController,
-    size: "modal-xs",
-    scope: $scope,
-    resolve: {
-      userForm: function() {
-        return $scope.userForm;
-      }
+    $scope.addBook = function(collection, book) {
+        Contents.addBook(collection, book)
+            .then(function() {
+                console.log('in remove book scope')
+                $scope.removeBook(book)
+            });
+        // $scope.newBook.title = "";
+    };
+
+    $scope.removeBook = function(book) {
+        Contents.removeBook("pending", {
+            title: book.title,
+            author: book.author
+        }).then(function() {
+            $scope.getPendingBooks()
+        });
+    };
+
+    $scope.getPendingBooks = function() {
+        console.log('in pending controller')
+        Contents.getBooks("pending")
+            .then(function(books) {
+                $scope.pending = books;
+                $scope.pendingCollection = [].concat(books);
+            });
     }
-  });
-};
 
-$scope.loadUser();
-// $scope.loadFriends();
+
+    // $scope.loadFriends = function() {
+    //   Contents.getFriends($scope.user.user_name)
+    //     .then(function(friends) {
+    //       console.log('FRIENDS:', friends)
+    //       $scope.friends = friends;
+    //     });
+    // };
+
+    $scope.getUsers = function() {
+        console.log('in get users')
+        Contents.getUsers()
+            .then(function(users) {
+                console.log('USERS:', users)
+                $scope.users = users
+            })
+    }
+
+    $scope.processFriend = function(friend) {
+        console.log('friend', friend)
+        Profile.processFriend(friend).then(function(resp) {
+            $state.go('friend')
+        })
+    }
+    $scope.photoOpen = function() {
+        var modalInstance = $uibModal.open({
+            templateUrl: "app/profile/photo.html",
+            controller: PhotoController,
+            size: "modal-xs",
+            scope: $scope,
+            resolve: {
+                userForm: function() {
+                    return $scope.userForm;
+                }
+            }
+        });
+    };
+    $scope.loadUser();
+    $scope.getPendingBooks();
+    // $scope.loadFriends();
+
+    $scope.aboutMeOpen = function() {
+        var modalInstance = $uibModal.open({
+            templateUrl: "app/profile/about-me.html",
+            controller: AboutMeController,
+            size: "modal-xs",
+            scope: $scope,
+            resolve: {
+                userForm: function() {
+                    return $scope.userForm;
+                }
+            }
+        });
+    };
+
+    $scope.photoOpen = function() {
```

Comment:
Please delete lines 103-118 then commit your changes. The pull request will get updated automatically